hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AtttributeResolver.kt

```diff
@@ -13,10 +13,28 @@ import kotlin.reflect.jvm.javaField
 
 typealias Attributes = Map<String, Attribute>
 
+val FUZZY_ATTRIBUTES = listOf(
+  "firstName",
+  "middleNames",
+  "lastName",
+  "prisonName",
+  "mostSeriousOffence",
+  "locationDescription",
+  "dischargeHospitalDescription",
+  "dischargeDetails",
+  "tattoos.comment",
+  "scars.comment",
+  "marks.comment",
+  "aliases.firstName",
+  "aliases.middleNames",
+  "aliases.lastName",
+)
```

Comment:
A fuzzy search is only available on attributes where it makes sense - generally free text entered by humans but also some of the longer descriptions.

It doesn't make sense to allow a fuzzy search on any static data such as reference codes or descriptions otherwise the search will turn up many false positives.